doc/quickstart/04_explorer.md

```diff
@@ -0,0 +1,41 @@
+# Explorer mode
+
+The node can be configured to work as a explorer. This consumes more resources, but makes it possible to query data otherwise not available.
+
+## Configuration
+
+There is two ways of enabling the explorer api. It can either be done by passing the `--enable-explorer` flag on the start arguemnts or by the config file: 
+
+``` yaml
+explorer:
+    enabled: true
+```
+
```

Comment:
add considerations about cors and link to the documentation of that too (i.e. in the REST config part of the doc it's supposed to be documented already)

Reply:
i.e : https://input-output-hk.github.io/jormungandr/configuration/network.html#rest-interface-configuration

Reply:
I added something about this, is it enough?